stratum/hal/bin/barefoot/docker/start-stratum-container.sh

```diff
@@ -23,6 +23,7 @@ else
     DOCKER_NET_OPTS="-p 28000:28000 "
     DOCKER_NET_OPTS+="-p 9339:9339 "
     DOCKER_NET_OPTS+="-p 9559:9559 "
+    DOCKER_NET_OPTS+="-p 9999:9999 "
```

Comment:
In the PDP sync we discussed about not exposing the port, but instead providing instructions to run telnet inside the container. The rationale was something around security, but I don't think I can provide a strong argument.

Still, I think it would be more convenient to allow running telnet inside the container.

For example, in Aether, we could access the BF shell in one command using kubectl exec, instead of forwarding ports and making sure telnet is installed.

If you agree, can you please update the README with an example of how to run telnet inside the container? We might also need to update the Dockerfile to make sure telnet is installed.